odbc/src/api/functions.rs

```diff
@@ -3475,7 +3570,10 @@ unsafe fn sql_set_env_attrw_helper(
                     SqlReturn::SUCCESS
                 }
                 None => {
-                    env_handle.add_diag_info(ODBCError::InvalidAttrValue("SQL_ATTR_ODBC_VERSION"));
+                    add_diag_info!(
```

Comment:
The macros in this function should be ` add_diag_with_function()` instead of the `add_diag_info!()`.

Reply:
ah, yes, I named the function, so this will work.
I did this at because I did not know if you would be Ok with hard-coding the parent SQL function name.
I hard-coded the function name now, to make it consistent with the other helper.
Let's do that for now and we can see later on if we want to use the helper names directly.